tree/30_generic_methods/variable_string_match.cf

```diff
@@ -0,0 +1,68 @@
+#####################################################################################
+# Copyright 2019 Normation SAS
+#####################################################################################
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, Version 3.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+################################################################################
+
+# @name Variable string match
+# @description Test the content of a string variable
+# @documentation Test a variable content and report a success if it matched, or an error if it does not or if the variable could not be found.
```

Comment:
Please add a mention to `condition_from_variable_match` which is useful want users only want a test and no errors.